src/rtext.c

```diff
@@ -1040,7 +1040,7 @@ void DrawTextEx(Font font, const char *text, Vector2 position, float fontSize, f
 
     int size = TextLength(text);    // Total size in bytes of the text, scanned by codepoints in loop
 
-    int textOffsetY = 0;            // Offset between lines (on line break '\n')
+    int textOffsetY = 0;            // Offset between lines (on linebreak '\n')
```

Comment:
rtext.c line 1043: The term in the C Language specifications is "newline" for the '\n' escape.  "linebreak" refers to any of the ways that one line is separated from the next and newline might be transformed into/from another in the stdio.h operations, depending on the computer platform.  But in the C program, it is `\n` (for code 0x0A always) with the Standard C library.  I know, fussy.

Reply:
Better to use newline then